roles/openstack/cleanup/compute/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+# Instances defaults
+
+instance_image: centos-minimal
```

Comment:
Oh sorry.
The file you specificly asked about is a file of default values.
For creating a server/instance you need a server name and image. Now what happens if user provided name but not an image? we can choose:
1. fail the run
2. use default image. using default image is risky but you might not have this default image in your cloud. I'm not aware of default common image that can be used here. we can still decide option 1 is better.